cwms-data-api/src/main/java/cwms/cda/data/dao/TimeSeriesDaoImpl.java

```diff
@@ -1,8 +1,7 @@
 package cwms.cda.data.dao;
```

Comment:
Nice organization. We should start slowing working this into the other DAOs

Reply:
Thanks.   Future change might be to make parameter object part of the method signatures in the interfaces and drop the methods with the huge number of parameters. Hopefully adding more options and filters will now be easier.